assets/src/edit-story/components/checklist/checks/pageBackgroundLowTextContrast/component.js

```diff
@@ -35,37 +46,45 @@ const PageBackgroundTextLowContrast = () => {
     () => filterStoryPages(story, pageBackgroundTextLowContrast),
     [story]
   );
+  const setHighlights = useHighlights(({ setHighlights }) => setHighlights);
+
+  const { title, footer } = ACCESSIBILITY_COPY.lowContrast;
+
   return (
     failingPages.length > 0 && (
       <ChecklistCard
-        title={__(
-          'Increase contrast between text and background color',
-          'web-stories'
-        )}
-        /* titleProps={{ onClick: () => { perform highlight here } }} */
-        footer={
-          <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
-            {__(
-              'Ensure legibility of text and ease of reading by increasing color contrast',
-              'web-stories'
-            )}
-            {
-              //       <Link
-              //         href={'#' /* figure out what this links to */}
-              //         size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
-              //       >
-              //         {'Learn more'}
-              //       </Link>
-            }
-          </Text>
+        title={title}
+        cardType={
+          failingPages.length > 1
+            ? CARD_TYPE.MULTIPLE_ISSUE
+            : CARD_TYPE.SINGLE_ISSUE
         }
-        /*
-        todo thumbnails for pages
+        footer={<DefaultFooterText>{footer}</DefaultFooterText>}
         thumbnailCount={failingPages.length}
-        thumbnail={<>
-            {failingPages.map(() => <Thumbnail />)}
-          </>}
-      */
+        thumbnail={
+          <>
+            {getActiveThumbnails(failingPages).map((page) => (
+              <Thumbnail
+                key={page.id}
+                onClick={() => {
+                  setHighlights({
+                    pageId: page.id,
+                  });
+                }}
+                type={THUMBNAIL_TYPES.PAGE}
+                displayBackground={
+                  <PagePreview
+                    page={page}
+                    width={THUMBNAIL_DIMENSIONS.WIDTH}
+                    height={THUMBNAIL_DIMENSIONS.HEIGHT}
+                    as="div"
```

Comment:
🤔 Should this be `forwardedAs`? I was looking at the `PagePreview` and does this prop get applied to the top level element or to the button?